pkg/cmd/pulumi/util.go

```diff
@@ -775,13 +775,25 @@ func (cancellationScopeSource) NewScope(events chan<- engine.Event, isPreview bo
 	return c
 }
 
+func makeJSONString(v interface{}) (string, error) {
+	out := bytes.NewBuffer([]byte{})
+	encoder := json.NewEncoder(out)
+	encoder.SetEscapeHTML(false)
+	encoder.SetIndent("", "  ")
+	err := encoder.Encode(v)
+	if err != nil {
+		return "", err
+	}
+	return out.String(), nil
+}
+
 // printJSON simply prints out some object, formatted as JSON, using standard indentation.
 func printJSON(v interface{}) error {
```

Comment:
Nit: perhaps put "indent" in the name of the func to further emphasize what it does.

Reply:
Since this function already existed and is used in many places in the CLI I'd rather not change them all here :)